packages/browser-wallet-api-helpers/CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## 0.2.1
+
+### Changed
+
+-   Fixed broken link + typos in README
+-   Removed parameters from smart contract types' payloads, due the wallet ignoring it in favor of separate arguments.
```

Comment:
This change will give type checking errors, so I think it should be a new major version.